task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```
@@ -22,6 +22,10 @@ spec:
   - description: Set cachi2 log level (debug, info, warning, error)
     name: log-level
     default: "info"
+  - description: |
+      Pass configuration to cachi2.
+      Note this needs to be passed as a YAML-formatted config dump, not as a file path!
+    name: config-file-content
```

```suggestion
    name: config-file-content
    default: ""
```
forgot we need a default value